app/models/related-item-contributor.js

```diff
@@ -9,20 +9,25 @@ const Validations = buildValidations({
     validator('presence', {
       presence: true,
       message: 'Related Item contributor must have a name',
-    })
+    }),
+    validator(function(value, options) { 
```

Comment:
Seems like we can remove this validator.
As far as I can tell, it just does a console log.

Reply:
Thanks - missed that when cleaning up the code :grimacing: